presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/MysqlDaoProvider.java

```diff
@@ -34,8 +33,10 @@ public MysqlDaoProvider(DbResourceGroupConfig config)
         requireNonNull(config, "DbResourceGroupConfig is null");
         MysqlDataSource dataSource = new MysqlDataSource();
         dataSource.setURL(requireNonNull(config.getConfigDbUrl(), "resource-groups.config-db-url is null"));
-        IDBI dbi = new DBI(dataSource);
-        this.dao = dbi.onDemand(ResourceGroupsDao.class);
+        this.dao = Jdbi.create(dataSource)
+                .installPlugin(new SqlObjectPlugin())
+                .open()
```

Comment:
Does the `Handle` need to be closed?

Reply:
Good catch, I copy/pasted this wrong from `H2DaoProvider` which was incorrect before. It should be `onDemand()` which automatically acquires and releases the handle.